pkg/interop/iterator/iterator.go

```diff
@@ -22,7 +22,7 @@ func Next(it Iterator) bool {
 // Value returns iterator's current value. It's only valid to call after
 // a successful Next call. This function uses `System.Iterator.Value` syscall.
 // For slices, the result is just value.
-// For maps, the result can be casted to a slice of 2 elements: a key and a value.
+// For maps, the result can be cast to a slice of 2 elements: a key and a value.
```

Comment:
https://english.stackexchange.com/questions/94565/can-casted-be-the-past-tense-of-cast
OK though.

Reply:
I like `casted` BTW, but Goland also reports the spelling error. Revert?